finetuner/tuner/dataset/helper.py

```diff
@@ -0,0 +1,15 @@
+import itertools
+from typing import TYPE_CHECKING, Any, Iterator
+
+from jina import DocumentArray
+
+if TYPE_CHECKING:
+    from ...helper import DocumentSequence
+
+
+def batch_document_sequence(docs: 'DocumentSequence', size: int):
+    """Batch a document sequence into DocumentArray batches"""
+    iterable: Iterator[Any] = iter(docs)
+    return iter(
+        lambda: DocumentArray(list(itertools.islice(iterable, size))), DocumentArray()
+    )
```

Comment:
i don't understand this function, why not `da.batch`? https://docs.jina.ai/fundamentals/document/documentarray-api/?highlight=da%20batch#batching

Reply:
because document sequence can be a list of documents

Reply:
how about convert it into a DA beforehead? I forgot why do we support list of Documents, seems now everyone is using the DA as input to finetuner @tadejsv

Reply:
okay

Reply:
I agree, let's wait for docarray to be released, then we can make this change